src/pickle-pricing.ts

```diff
@@ -0,0 +1,50 @@
+import { getAddress } from "@ethersproject/address";
+import { BigNumber } from "bignumber.js";
+
+import { Address } from "./types/common";
+
+const hour = 1000 * 60 * 60;
+const pickleApiUrl = "https://stkpowy01i.execute-api.us-west-1.amazonaws.com/prod/protocol/pools";
```

Comment:
`const` name should be PascalCase

```suggestion
const HourInMilliseconds = 1000 * 60 * 60;
const PickleApiUrl = "https://stkpowy01i.execute-api.us-west-1.amazonaws.com/prod/protocol/pools";
```